prime-select.sh

```diff
@@ -219,12 +232,12 @@ function apply_current {
             # this can happen if user set nvidia but changed to "Integrated only" in BIOS (possible on some MUXED Optimus laptops)
             # in that case the NVIDIA card is not visible to the system and we must switch to intel
             
-            logging "Forcing intel due to NVIDIA card not found"
-            current_type="intel"
+            logging "Forcing intel due to NVIDIA card not found" current_type="intel"
         fi
         
         
         set_$current_type
+        ask_to_logout
```

Comment:
Doesn't this change a previously non-interactive script to an interactive script?

Reply:
Yes, after asking prime-select to switch for a different from driver, it will ask you if you wish to logout. 

By default, if you press enter, it won't log you out.

Reply:
Since users may used to use the script in a non-interactive way, I don't want to see such a change honestly. If, then only with an additional option "[--interactive]" or whatever.